src/NHSD.BuyingCatalogue.Ordering.Api/Controllers/CatalogueSolutionsController.cs

```diff
@@ -12,11 +16,30 @@ namespace NHSD.BuyingCatalogue.Ordering.Api.Controllers
     [Authorize(Policy = PolicyName.CanAccessOrders)]
     public sealed class CatalogueSolutionsController : ControllerBase
     {
+        private readonly IOrderRepository _orderRepository;
+
+        public CatalogueSolutionsController(IOrderRepository orderRepository)
+        {
+            _orderRepository = orderRepository ?? throw new ArgumentNullException(nameof(orderRepository));
+        }
+
         [HttpGet]
-        public ActionResult GetAll()
+        public async Task<ActionResult> GetAllAsync(string orderId)
```

Comment:
Might be a good idea to specify the return type E.g. Task<ActionResult<CatatalogueSolutionsModel>>
I know this would probably change your tests slightly but is a good idea

Reply:
*With the return type